platform/linux/linux.cmake

```diff
@@ -184,6 +184,9 @@ target_link_libraries(
 # Bundle system provided libraries
 if(MLN_CORE_INCLUDE_DEPS AND NOT MLN_USE_BUILTIN_ICU AND NOT "${ARMERGE}" STREQUAL "ARMERGE-NOTFOUND")
     message(STATUS "Found armerge: ${ARMERGE}")
+    include(${CMAKE_CURRENT_LIST_DIR}/cmake/find_static_library.cmake)
+    find_static_library(PNG_STATIC_LIB NAMES png)
```

Comment:
could we add the other image libraries as well?

- libjpeg
- libwebp

If possible via this route (this has not caused an issue for now => I would be happy to go at this one issue at a time if you would be okay with revisiting this 😅):
- libz
- libcurl4-openssl-dev
- libsqlite3
- libuv1

Reply:
Doing them in one go sounds better.

Reply:
If you think that they are risky, we can also move them to when we might need them.

Reply:
actually, sqlite and openssl are not even public symbols 🎉

Reply:
added the whole bunch, let's see